### ISSUE TITLE
accessibility issue 26 (medium) - hover color contrast for buttons in you-searched-for section

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#245]
The color contrast of the red background (that appears on hover) with the blue X remove link on the “you searched for” filters is insufficient. The color contrast should be 3:1 for all user interface components.

**Comment by yulgit1** [comment]
resolved:https://github.com/ycba-cia/blacklight-collections2/commit/560caac1a456264071bee7dcaf113dd176e8284c
